include/lsst/jointcal/Frame.h

```diff
@@ -28,8 +28,8 @@ class Frame {
 
   //! this one is dangerous: you may swap the 2 middle arguments.
   //! Prefer next one
-  Frame(const double &xMin, const double &yMin,
-	const double &xMax, const double &yMax);
+  Frame(double xMin, double yMin,
+	double xMax, double yMax);
```

Comment:
Sorry, I was referring to the comment below these lines, about how this invocation of Frame was potentially confusing (which I agree with). I was asking if you could refactor the code to eliminate this syntax.